pom.xml

```diff
@@ -116,7 +116,7 @@
   </scm>
 
   <properties>
-    <java-build>1.6</java-build>
+    <java-build>1.7</java-build>
```

Comment:
this change is not related to issue reported (anyway covered in your other pull request), please revert